### DISCUSSION TITLE
how to get the output values with `wav` as output waveform by synthesize function to wav file?

### DISCUSSION THREAD

**Discussion by tanhzongbei** [coqui-ai/TTS#3379] [General Q&A]
how to get the output values with `wav` as output waveform by synthesize function to wav file?

 outputs = model.synthesize(
        request.input,
        config,
        speaker_wav=sample_voice_path,
        gpt_cond_len=3,
        language="en",
    )

how to save outputs['wav'] to wav file?

**Comment by erogol** [comment]
Use TTS API for that. See the docs.

**Reply by tanhzongbei** [reply]
no, I want to use the model derictly！
I have sloved it

**Reply by tanhzongbei** [reply]
numpy_transforms.save_wav(wav=wav, path=out, sample_rate=24000) is what I want

**Comment by tanhzongbei** [comment]
OK I have solved it

**Reply by storuky** [reply]
Hey, how did you solve this?

**Comment by realpug** [comment]
from TTS.utils.audio.numpy_transforms import save_wav

save_wav(wav=outputs['wav'], path='output.wav', sample_rate=24000)
# fixed it for me..
